maven-release-plugin/src/it/projects/prepare/MRELEASE-1053/verify.groovy

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+// check parent project
+def project = new XmlSlurper().parse( new File( new File ( basedir, 'module-parent' ), 'pom.xml.next' ) )
+assert project.version.text() == '1.1-SNAPSHOT'
+assert project.scm.url.text() == 'http://localhost/myscm/${project.scm.tag}'
+
+// check execution root project
+project = new XmlSlurper().parse( new File ( basedir, 'pom.xml.next' ) )
+assert 0 == project.scm.size() // scm element must not exist in reactor root
+
+// check module a  project
+project = new XmlSlurper().parse( new File( new File ( basedir, 'module-a' ), 'pom.xml.next' ) )
+assert 0 == project.scm.size() // scm element must not exist in reactor root
+return true;
```

Comment:
Groovy implicitly return the last result, so `return true`  can be removed.

Reply:
Fixed in https://github.com/apache/maven-release/commit/42eeaf70dafedc759e7bd5421842d2ad687f7deb.